frontendservice/src/views/ShopOwnerHome.vue

```diff
@@ -1,79 +1,171 @@
+            <td v-if="shop['shop'].status=='ACTIVE'">
+              <button @click="deleteshop(shop['shop'].shopId)">x</button>
+            </td>
+            <td v-if="shop['shop'].status!='ACTIVE'">
+              <button>restore</button>
+            </td>
+          </tr>
+        </tbody>
+      </table>
     </div>
     <div v-if="isLoggedIn && !isVerified">
-      <router-link to="/verify">Verify Me!</router-link>
-    </div>
-    <div v-if="!isLoggedIn">
-      <router-link to="/about">Back</router-link>
+      <!-- TODO: ikoder - Add other features for this page -->
+      <router-link to="/verify"><button class="button is-primary">Verify Me!</button></router-link>
     </div>
-    <br />
-    <a class="button is-danger" v-if="isLoggedIn" @click="logout">Logout</a>
   </div>
 </template>
 
+<script src="https://cdn.jsdelivr.net/npm/sweetalert2@9"></script>
 <script>
+import shopOwnerNavbar from "../components/NavBar-ShopOwner";
+import axios from "axios";
+import Swal from "sweetalert2";
 import firebase from "firebase";
-
 export default {
-  name: "shopOwnerHome",
-
+  name: "MyShops",
+  components: {
+    shopOwnerNavbar
+  },
   data() {
     return {
-      name: "",
-      uid: "",
+      shopList: [],
+      searchResult: "",
+      renderComponent: true,
       isLoggedIn: false,
-      isVerified: false,
+      isVerified: false
     };
   },
+  methods: {
+    deleteshop: function(id) {
+      Swal.fire({
+        title: "Are you sure?",
+        text: "",
+        icon: "warning",
+        showCancelButton: true,
+        confirmButtonColor: "#3085d6",
+        cancelButtonColor: "#d33",
+        confirmButtonText: "Yes, delete it!"
+      }).then(result => {
+        if (result.isConfirmed == true) {
+          axios
+            .put("http://penguin.termina.linux.test:8080/shop/", {
+              shopId: id,
+              shopStatus: "DELETED"
+            })
+            .then();
+          this.renderComponent = false;
+          var self = this;
+          // TODO : ikoder - Add error handling in case of server error.
+          this.$nextTick(() => {
+            axios
+              .get(
+                "http://penguin.termina.linux.test:8080/shop/" +
+                  firebase.auth().currentUser.uid
+              )
+              .then(response => (self.shopList = response.data.shops));
+            self.renderComponent = true;
+          });
+        }
+      });
+    }
+  },
   created() {
     var user = firebase.auth().currentUser;
     if (user) {
-      this.name = user.displayName;
-      this.uid = user.uid;
       this.isLoggedIn = true;
       if (user.emailVerified) {
         this.isVerified = true;
       }
     }
+    var self = this;
+    setInterval(function() {
+      axios
+        .get(
+          "http://penguin.termina.linux.test:8080/shop/" +
+            firebase.auth().currentUser.uid
+        )
+        .then(response => (self.shopList = response.data.shops));
+    }, 2000);
```

Comment:
Can you mention the unit of time in comment beside 2000 to improve readability?
Like,  2000 /* milliseconds */

Reply:
added!